packages/solana/lib/src/rpc_client/signature_status.dart

```diff
@@ -33,11 +32,37 @@ class SignatureStatus {
 }
 
 @JsonSerializable(createToJson: false)
-class SignatureStatusesResponse
-    extends JsonRpcResponse<ValueResponse<Iterable<SignatureStatus?>>> {
-  SignatureStatusesResponse(ValueResponse<Iterable<SignatureStatus?>> result)
-      : super(result: result);
+class SignatureStatusesResponse {
+  SignatureStatusesResponse({required this.result});
 
   factory SignatureStatusesResponse.fromJson(Map<String, dynamic> json) =>
       _$SignatureStatusesResponseFromJson(json);
+
+  final _SignatureStatusesResult result;
+}
+
+@JsonSerializable(createToJson: false)
+class _SignatureStatusesResult {
+  _SignatureStatusesResult({required this.value});
+
+  factory _SignatureStatusesResult.fromJson(Map<String, dynamic> json) =>
+      _$SignatureStatusesResultFromJson(json);
+
+  @_NullableListConverter()
```

Comment:
What's so special about this list, why does need a custom converter?

Reply:
The generated `FromJson` for a `List<Something?>` does not take into account the fact that the template parameter is nullable. Leading to an exception about trying to convert `null` to `Something` (or to `Map<String, dynamic>`, I can't remember which exception is thrown).

Reply:
This was the tricky thing I spoke about in the call.